packages/dashboards/package.json

```diff
@@ -21,60 +21,59 @@
     "test": "export COVERAGE=true; ember test --test-port 0"
   },
   "dependencies": {
-    "ember-cli-babel": "^6.11.0",
-    "ember-cli-htmlbars": "^2.0.3",
-    "ember-math-helpers": "^2.3.0"
-  },
-  "devDependencies": {
-    "broccoli-asset-rev": "^2.4.5",
     "ember-ajax": "^3.0.0",
     "ember-basic-dropdown-hover": "^0.3.0",
     "ember-c3": "0.2.3",
-    "ember-cli": "~2.16.2",
+    "ember-cli-babel": "^6.11.0",
     "ember-cli-bootstrap-datepicker": "^0.5.3",
     "ember-cli-clipboard": "^0.8.1",
-    "ember-cli-code-coverage": "^0.4.1",
-    "ember-cli-dependency-checker": "^2.1.1",
-    "ember-cli-eslint": "^4.0.0",
     "ember-cli-format-number": "^2.0.0",
-    "ember-cli-htmlbars-inline-precompile": "^1.0.0",
-    "ember-cli-inject-live-reload": "^1.4.1",
-    "ember-cli-less": "^1.5.5",
-    "ember-cli-mirage": "~0.2.9",
-    "ember-cli-moment-shim": "3.5.0",
-    "ember-cli-qunit": "^4.2.1",
-    "ember-cli-shims": "^1.1.0",
-    "ember-cli-sri": "^2.1.0",
+    "ember-cli-htmlbars": "^2.0.3",
     "ember-cli-string-helpers": "^1.6.0",
-    "ember-cli-uglify": "^2.0.0",
     "ember-collection": "1.0.0-alpha.7",
     "ember-composable-helpers": "^2.1.0",
     "ember-cp-validations": "^3.5.0",
-    "ember-data": "~2.12.2",
     "ember-data-model-fragments": "~2.11.5",
-    "ember-disable-prototype-extensions": "^1.1.2",
-    "ember-export-application-global": "^2.0.0",
     "ember-font-awesome": "3.0.5",
     "ember-get-config": "0.2.4",
     "ember-gridstack": "1.2.0",
-    "ember-load-initializers": "^1.0.0",
     "ember-lodash": "^4.17.5",
     "ember-math-helpers": "^2.3.0",
     "ember-modal-dialog": "^2.4.1",
     "ember-moment": "7.5.0",
     "ember-power-select": "^1.10.4",
     "ember-promise-helpers": "^1.0.3",
-    "ember-resolver": "^4.0.0",
     "ember-route-action-helper": "^2.0.6",
     "ember-sortable": "^1.10.0",
-    "ember-source": "~2.16.0",
     "ember-tag-input": "^1.2.1",
     "ember-tether": "^0.4.1",
     "ember-toggle": "5.2.1",
     "ember-tooltips": "^2.10.0",
     "ember-truth-helpers": "^2.0.0",
     "ember-uuid": "^1.0.0",
-    "liquid-fire": "^0.29.2",
+    "liquid-fire": "^0.29.2"
+  },
+  "devDependencies": {
+    "broccoli-asset-rev": "^2.4.5",
+    "ember-cli": "~2.16.2",
+    "ember-cli-code-coverage": "^0.4.1",
+    "ember-cli-dependency-checker": "^2.1.1",
+    "ember-cli-eslint": "^4.0.0",
+    "ember-cli-htmlbars-inline-precompile": "^1.0.0",
+    "ember-cli-inject-live-reload": "^1.4.1",
+    "ember-cli-less": "^1.5.5",
+    "ember-cli-mirage": "~0.2.9",
+    "ember-cli-moment-shim": "3.5.0",
+    "ember-cli-qunit": "^4.2.1",
+    "ember-cli-shims": "^1.1.0",
+    "ember-cli-sri": "^2.1.0",
+    "ember-cli-uglify": "^2.0.0",
+    "ember-data": "~2.12.2",
```

Comment:
Is there a reason that our ember-data version is out of sync with ember-source?

Reply:
There was.. I can't remember it though.